t/scripts/unxz

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+opt_k=$1
+opt_f=$2
+file=$3
+newfile=${file%.xz}
+mv "$file" "$newfile"
```

Comment:
Instead of adding a new file IMHO would be nicer to overwrite the command called within the actual code. Or make it a configurable variable that you overwrite from tests

Reply:
done